### ISSUE TITLE
WCAG

### ISSUE THREAD

**Issue by giogardin** [ggardin/tecweb#322]
Raccolta di errori, errori probabili, avvisi sollevati da TotalValidator.

- [x] 404.html
	- nessun errore

- [x] 500.html
	- nessun errore

- [ ] about.html
	- [ ] P871

- [ ] aggiungi.html
	- [ ] P871

- [ ] cerca.html
	- [ ] P871
	- [ ] W872

- [x] collezione.html
	- nessun errore

- [ ] dati_film.html
	- [ ] P871

- [ ] dati.html
	- [ ] P871

- [ ] film.html
	- [ ] P871

- [ ] gest_collezione.html
	- [ ] P871

- [ ] gest_film.html
	- [ ] P871

- [ ] gest_list.html
	- [ ] P871

- [ ] gest_persona.html
	- [ ] P871

- [x] index.html
	- nessun errore

- [ ] list.html
	- [ ] P871

- [ ] lists.html
	- [ ] P871

- [ ] login.html
	- [ ] P871

- [ ] persona.html
	- [ ] P871

- [ ] signup.html
	- [ ] P871

- [ ] stats.html
	- [ ] P871

- [ ] user.html
	- [ ] P871

Errori segnalati (falsi positivi) da discutere in relazione:

1. P871 - SC 1.1.1/2.4.4/2.4.9: Link text is missing. Describe the purpose of a link by providing descriptive text as the content of the element, or provide an 'aria-label' or 'aria-labelledby' attribute. The description lets a user distinguish this link from other links in the Web page and helps the user determine whether to follow the link. The URL of the destination is generally not sufficiently descriptive. See [WCAG 2.1 HTML Technique H30](https://www.w3.org/WAI/WCAG21/Techniques/html/H30.html).

1. W872 - SC 3.2.2: Using 'onchange' with selections may prevent keyboard operation. Using 'onchange' with selections may prevent keyboard operation, because the handler fires as the selection is changed using the keyboard, which may move the focus or change the context. See [Understanding 3.2.2](https://www.w3.org/WAI/WCAG21/Understanding/on-input.html).
